docs/config.json

```diff
@@ -506,6 +506,10 @@
               "title": "AWS ElastiCache & MemoryDB",
               "slug": "/database-access/guides/redis-aws/"
             },
+            {
+              "title": "AWS DynamoDB",
```

Comment:
This seems like an Application Service guide—should it go in that section instead?